tests/caasp/filesystem_ro.pm

```diff
@@ -14,16 +14,17 @@
 use base "opensusebasetest";
```

Comment:
I guess if you can reuse the module outside the context of "caasp" please also move it out of the "caasp" subfolder. Maybe we can have a new folder "transactional" which is used by the corresponding roles, caasp, microos and kubic?